sdcm/nemesis.py

```diff
@@ -258,7 +258,7 @@ def get_node_info_list(verification_node):
                 self._terminate_cluster_node(self.target_node)
                 # Replace the node that was terminated.
                 if add_node:
-                    self._add_and_init_new_cluster_node()
+                    self._add_and_init_new_cluster_node(timeout=10800)
```

Comment:
@juliayakovlev , please add a comment that explains your change.
something like:
"When adding node after decommission the node is declared as up only after it completed bootstrapping, increasing the timeout for now"

Reply:
We also need to open a ticket as a todo, to be able to query the node status via API (either it's bootstraping, leaving, etc...).
Then, we can make it better than just waiting for TO.